### ISSUE TITLE
Very high NDV API call rate

### ISSUE THREAD

**Issue by sergeykad** [stefanneuhaus/dependencycheck-central-mysql-docker#23]
[NDV API Documentation ](https://nvd.nist.gov/general/news/API-Key-Announcement)recommends calling the API no more than once every two hours, while the container calls it every two minutes. It can create a DDoS attack if enough people use this project.

There is also no easily accessible way to configure it. Exposing the cron configuration as an environment variable can be helpful.

**Comment by nauni77** [comment]
Yes, thats true. I changed some things and added a pull request, but nobody take care. So I released my fork to make it work for my purposes:

https://hub.docker.com/repository/docker/nauni1977/owasp-db-cache/general

- database will be reused and not build at every start of the container; drop it if SQL-Files change
- you can define the call rate (default 1 per hour)

I use normally the command to start:
```
docker run --rm --name owasp-db -p 3306:3306 -v owasp-db:/var/lib/mysql -v owasp-config:/var/lib/owasp-db-cache -e NVD_API_KEY=<add your api key> -e UPDATE_INTERVALL="0/5 * * * *" nauni1977/owasp-db-cache:10.0.1
```

If you are interessted I will add some documentation - for now it's only used by myself.

If the project is interessted to add my changes - please tell me and I will add a new pull request.

Best regards, Oli

**Comment by sergeykad** [comment]
Fixed by https://github.com/stefanneuhaus/dependencycheck-central-mysql-docker/commit/928075b587b4ec37b4d11e45f5211c41879fb597
